airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/source.py

```diff
@@ -523,21 +523,20 @@ def get_authenticator(config: Mapping) -> Oauth2Authenticator:
         if config.get("credentials_json"):
             return GoogleAnalyticsServiceOauth2Authenticator(config)
 
-        auth_params = config.get("credentials")
+        auth_params = config["credentials"]
```

Comment:
`credentials` is not a required field. We should leave `config.get("credentials")` for supporting old config version. We haven't the `credentials` field before OAuth implementation.
Same for other fields below.